src/server.py

```diff
@@ -361,6 +361,14 @@ def finished(self):
 
                 output_logger.close()
 
+                # What's the diffrence between "get_config()" and "config" ?
+                # process_wrapper.get_config().output_files = file_download_feature.update_output_files_vars_with_args(
+                #    process_wrapper.get_config().output_files,
+                #    process_wrapper.execution_info.param_values)#
+                process_wrapper.config.output_files = file_download_feature.update_output_files_vars_with_args(
```

Comment:
could you move this call inside _prepare_downloadable_files_? So something like:
```
def prepare_downloadable_files(...):
    output_files = config.output_files

    if not output_files:
        return []
    
    output_files = update_output_files_vars_with_args(...)
```